src/home/views.py

```
@@ -19,3 +19,33 @@
+
+    # Problema 2: Lista invece di comprehension
+    result_posts = []
+    for post in posts_data:
+        if post["active"] == True:  # Dovrebbe essere 'is True'
+            result_posts.append(post["title"])
+
+    # Problema 3: String concatenation invece di f-string
+    message = "User " + user.username + " has " + str(len(result_posts)) + " posts"
+
+    # Problema 4: Doppia negazione
+    if not user.is_active != True:
```

Flake8 (7.2.0)

[error] comparison to True should be 'if cond is not True:' or 'if not cond:'

(E712)